actors/runtime/src/runtime/fvm.rs

```diff
@@ -149,6 +151,14 @@ where
         fvm::actor::get_actor_code_cid(addr)
     }
 
+    fn is_builtin_actor(&self, code_id: &Cid) -> Option<Type> {
+        fvm::actor::is_builtin_actor(code_id)
```

Comment:
nit: `resolve_builtin_actor_type`or just `builtin_actor_type`? `is_` methods usually return a boolean.

Reply:
Yep, I thought of this but concluded that returning an Option was already a higher-level version of returning a boolean. I'll change it, though.